### PR TITLE
Added tidyverse example

### PR DESCRIPTION
**Pull request by edwindj**
Added an example for building a tidyverse image. 
Note that it uses the latest github version of readxl, since the CRAN version (1.3.1) is not compatible with alpine.

### CODE REVIEW

File: examples/tidyverse/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+FROM rhub/r-minimal
+
+RUN installr -d \
+   -t "R-dev file automake autoconf linux-headers" \
+   -a "libxml2-dev icu-libs" `# needed for xml2 and stringr`  \
```
Comment (gaborcsardi) [inline examples/tidyverse/Dockerfile:6]: I don't think we need libxml2-dev on the image, just the lib itself, so you can add `libxml2-dev` to the `-t` line and have `libxml` in `-a`.

Comment (edwindj) [reply examples/tidyverse/Dockerfile:6]: Ah, thanks, will do!

**Comment by edwindj** [comment]
Should be fine now. (Works on my machine)

**Comment by gaborcsardi** [comment]
Thanks a lot!

**Pull request merged**